src/utils/file_upload.rs

```diff
@@ -263,7 +275,11 @@ fn upload_files_chunked(
     progress.finish_and_clear();
 
     if response.state.is_pending() {
-        println!("{} File upload complete", style(">").dim());
+        if context.wait {
+            bail!("Failed to process files in {}s", max_wait.as_secs());
+        } else {
+            println!("{} File upload complete", style(">").dim());
```

Comment:
Maybe we could say something like `upload complete (processing pending)` or something of that sort? As this is what is actually happening?

Reply:
How about:

```suggestion
            println!("{} File upload complete (processing pending on server)", style(">").dim());
```

Reply:
Sounds good to me!